### DISCUSSION TITLE
VID/PID hardcoded? (config in menu request)

### DISCUSSION THREAD

**Discussion by paudrat** [hmatuschek/qdmr#486] [General]
Hello,
I am trying to connect a Baofeng DR1801UV.
At the command: **dmrconf -V -D ttyUSB0 -R DR1801UV detect**
i get: **Debug in cli/autodetect.cc@40: Autodetect radios.
Debug in lib/usbserial.cc@162: Search for serial port with matching VID:PID 28e9:18a.
Debug in lib/usbserial.cc@162: Search for serial port with matching VID:PID 1fc9:94.
Debug in lib/hid_libusb.cc@128: Search for HID interfaces matching VID:PID 15a2:73.
Debug in lib/dfu_libusb.cc@156: Search for DFU devices matching VID:PID 483:df11.**
My usb/serial adapter has a different VID/PID.
Is it possible to choose the usb adapter in a menu?
Many thanks for your job.
Best regards.
F4DWC

**Comment by hmatuschek** [comment]
Yes you can by specifying the interface to use.

- First, find the serial port, the radio is connected to. eg. /dev/ttyUSB0
- Then use `-D` or `--device` to specify the device and the `-R` or `--radio` options to specify the radio.

``` 
 dmrconf --verbose read --device=ttyUSB0 --radio=dr1801uv codeplug.yaml
```
Detecting makes no sense, as the device cannot be detected reliably. However, you should be able to handle the radio (read/write) by specifying the interface.

**Reply by paudrat** [reply]
many thanks

**Comment by hmatuschek** [comment]
See #485.
